starwars/src/components/CardInfo.js

```diff
@@ -0,0 +1,22 @@
+import React from  'react'
+import Card from './Card';
+import './StarWars.css'
+
+class CardInfo extends React.Component {
+
+    render() {
+        return (
+            <div className="cardInfo">
+                {this.props.starWarsInfo.map(obj => {
+                    return (
+                        <div key={obj.created}>
```

Comment:
Nice work on mapping over the data. My recommendation would be to put the div inside of the component you are returning, to keep the styling compartmentalized. And then apply the key to the component being returned. This is personal preference though.